WcaOnRails/app/webpacker/packs/auto_numeric.js

```diff
@@ -1 +1 @@
-import '../javascript/auto-numeric';
+import '../lib/auto-numeric';
```

Comment:
Do we need this file?

Reply:
From my understanding, these one-liners are "entry points" generated by Webpack, so that the old, not-so-fancy part of Javascript that lives under `WcaOnRails/assets/javascripts` can access stuff that Webpack has compiled.

Reply:
That being said, for this PR I just followed the mantra of "never touch a running system". Meaning I didn't contemplate on why these files were there, I just assumed they are there for a reason and the folder rename is "collateral damage" from my React folder changes.